golang-basic/00-pre-requirements.md

```diff
@@ -0,0 +1,5 @@
+## Pre requirements
```

Comment:
I guess we can make it an md book that they can access and serve from their local.

Reply:
yes, or add a new tab ( `Docs`) to this project and host it by ourself